qt/scientific_interfaces/Indirect/IndirectLoadILL.cpp

```diff
@@ -124,12 +186,25 @@ void IndirectLoadILL::run() {
                "'"
                ",'" +
                plot + "'," + save + ")";
+    runPythonScript(pyInput);
+#else
+    emit showMessageBox("IN16B is currently the only instrument supported in "
```

Comment:
Do you know why we don't support anything but IN16B, wouldn't there be legacy data to support?

Reply:
Getting workbench to support them will take some work. This is low priority and so can be implemented later on.

Reply:
Cool :+1: